src/web/app/components/question-responses/grouped-responses/grouped-responses.component.ts

```diff
@@ -22,6 +22,7 @@ export class GroupedResponsesComponent extends ResponsesInstructorCommentsBase i
   CommentRowMode: typeof CommentRowMode = CommentRowMode;
 
   @Input() responses: QuestionOutput[] = [];
+  @Input() userToEmail: Record<string, string> = {};
```

Comment:
Not used?

Reply:
It's used in the template for `grouped-responses.component`